src/languages/html.coffee

```diff
@@ -6,41 +6,141 @@ parse5 = require 'parse5'
 ATTRIBUTE_CLASSES = ['#attribute']
 
 COLORS = {
-  'Default': 'cyan',
-  '#comment': 'grey'
-  'a': 'grey',
-  'b': 'teal',
-  'body': 'return',
-  'br': 'command',
-  'button': 'yellow',
-  'center': 'red',
-  'div': 'amber',
-  'document': 'bluegrey',
-  'font': 'value',
-  'form': 'deeporange',
-  'h1': 'teal',
-  'h3': 'indigo',
-  'head': 'cyan',
-  'hr': 'lime',
-  'html': 'amber',
-  'iframe': 'green',
-  'img': 'green',
-  'input': 'brown',
-  'label': 'lightblue',
-  'li': 'pink',
-  'link': 'purple',
-  'marquee': 'command',
-  'meta': 'error',
-  'option': 'control',
-  'p': 'deeppurple',
-  'script': 'orange',
-  'select': 'indigo',
-  'strong': 'yellow',
-  'table': 'lightgreen',
-  'td': 'lightblue',
-  'title': 'green',
-  'tr': 'bluegrey',
-  'ul': 'blue'
+  #Metadata
```

Comment:
Let's go ahead and make this more similar to the configuration in coffee.coffee.

  * Make it do an indirection through a "categories" option that you can override in modeOptions.  So that you can change the color of whole categories by just changing modeOptions.categories.

  * For each tag, instead of just specifying a color, specify a whole object that in the future can include a dropdown option, like  `{ category: 'content', dropdown: [['href=""', 'name=""']] }`.

  * Don't worry about adding the dropdown option here yet, but that would be the next thing to do in a future PR - based on attribute statistics, add the most common attributes as choices.